static/js/map.js

```diff
@@ -0,0 +1,251 @@
+// const modisURL = "modis.csv";
+const modisURL = "https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_modis";
+const viirsURL = "https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_viirs";
+const states = "https://raw.githubusercontent.com/rowanhogan/australian-states/master/states.geojson";
+const areaURL = "http://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_bystate";
+
+// TODO load in second dataset, create function that allows them to choose which dataset to use
+
+//////////// IMPORT THE DATA //////////////////
+function getData(modisURL, viirsURL) {
```

Comment:
This is fine calling d3.json multiple times. JavaScript does have a Promise.all() method that makes making multiple api calls easier/cleaner. See the section on loading multiple files: http://learnjsdata.com/read_data.html

No need to make any changes now... just something to take note...